src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -32,10 +32,6 @@
     <IntermediateRefAssembly Remove="@(IntermediateRefAssembly)" />
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
Just to make sure I understand, this PR is meant to fix issues when running on MSBuild. If you remove this block, does it work for both MSBuild and VS? Or does changing this break VS?

Reply:
Hi @jeffrosenberg - the problem I am seeing is that the creation script is generated fine but only actually copied to output in VS - when run on TeamCity (using MSBuild), for some reason the sql file is not progressing beyond the `obj` folder to the target output folder.

I *think* that this is because the "SqlArtifacts" `itemgroup` is not updated after the sql is generated by the CoreCompile... it seems that if we use CreateItem then it does pick up the new file.

Could be that @jmezach will have some thoughts on this as I think it was he that suggested I use the ItemGroup ...

Reply:
It seems that MSBuild is evaluating the `ItemGroup` once and then doesn't update it again. I guess in that scenario `CreateItem` makes sense.